src/utils/finders.js

```diff
@@ -46,7 +46,7 @@ function getDependencies(filename, basedir) {
   const localFilesToProcess = [filename]
 
   function handle(name, basedir, optionalDependencies) {
-    const moduleName = requirePackageName(name.replace(/\\/, '/'))
+    const moduleName = requirePackageName(name.replace(/\\/g, '/'))
```

Comment:
BTW if this was supposed to change only the first occurrence maybe it should be `'\'` and not a regex